docs/content/_index.md

```diff
@@ -1,12 +1,15 @@
 ---
 title: "Overview"
+weight: 20
```

Comment:
The weights aren't working as I'd expect... but they seem to be showing up in a reasonable order? 🤷 WFM.